src/components/_modal/pages/receipt/Receipt.page.js

```diff
@@ -169,15 +170,36 @@ export function Receipt({
               {Translate({ context: "receipt", label: "errorOccured" })}
             </Title>
             {hasFailed && failedMessage && (
-              <Text
-                tag="div"
-                type="text2"
-                className={styles.errorText}
-                dataCy="order-failed-message"
-              >
+              <Text tag="div" type="text2" dataCy="order-failed-message">
                 {failedMessage}
               </Text>
             )}
+            {hasFailed && (
+              <span className={styles.contactDBC}>
```

Comment:
Kunne man bruge div istedet for span her?

Reply:
hmm... det er en inline, hvorfor er div bedre her?